doc/sphinx-guides/source/developers/dataset-migration-api.rst

```diff
@@ -31,7 +31,7 @@ To import a dataset with an existing persistent identifier (PID), the provided j
   
   curl -H X-Dataverse-key:$API_TOKEN -X POST $SERVER_URL/api/dataverses/$DATAVERSE_ID/datasets/:startmigration --upload-file dataset-migrate.jsonld
 
-An example jsonld file is available at :download:`dataset-migrate.jsonld <../_static/api/dataset-migrate.jsonld>` . Note that you would need to replace the PID in the sample file with one supported in your Dataverse instance.
+An example jsonld file is available at :download:`dataset-migrate.jsonld <../_static/api/dataset-migrate.jsonld>` . Note that you would need to replace the PID in the sample file with one supported in your Dataverse instance. You also need to replace the dataverse.siteUrl in jsonld @context with your current Dataverse site URL. 
 
```

Comment:
I'm just putting this comment at the bottom.

@mjlassila from our [conversation](https://dataverse.zulipchat.com/#narrow/channel/378866-troubleshooting/topic/Metadata.20format.20for.20Dataset.20Migration.20API.20in.206.2E5/near/494778081) I was sort of hoping you'd include your BaseX and XQuery script at https://gist.github.com/mjlassila/ecdbd11447ccdf87995db20bfc5e686c 😄 . It it worth writing up and including? (I don't even know how to run it but I'm happy to learn!)